scripts/breaking_changes_checker/breaking_changes_tracker.py

```diff
@@ -468,6 +468,7 @@ def check_positional_parameter_removed_or_renamed(
                     )
 
     def check_class_instance_attribute_removed_or_renamed(self, components: Dict) -> None:
+        deleted_props = []
         for prop in components.get("properties", []):
             if isinstance(prop, jsondiff.Symbol):
                 deleted_props = []
```

Comment:
Do we still need the `deleted_props` on this line?

Reply:
removed, thanks!